### ISSUE TITLE
Example of a service using pleaserun

### ISSUE THREAD

**Issue by nukepuppy** [greglu/dropwizard-cookbook#6]
HI there, 
Since init_script_source is now removed.. it would be good to see an example of using pleaserun from chef to get a service running..

**Comment by nukepuppy** [comment]
anyone around?

**Comment by greglu** [comment]
Sorry this issue got lost amongst my notifications.

The `init_script_source` was previously used to pass in a custom init script. However, with the integration of [pleaserun](https://github.com/jordansissel/pleaserun), the idea is to not require building your own init scripts anymore. Instead, `pleaserun` should autodetect your platform's init system and create the scripts for you.

What kinds of examples are you interested in seeing that aren't covered in the [README.md#Examples](https://github.com/greglu/dropwizard-cookbook#examples) section?

**Comment by nukepuppy** [comment]
Heya I'll definitely read thru examples again..  but I think ideally at least provide an example of how to reach parity with the previous behavior using 'pleaserun' .. this way folks can extend it or configure it to be Non breaking..  and add to it. 

the way it is now..  it's a breaking change..

**Comment by nukepuppy** [comment]
I'm currently looking to previois version mostly due to cycles..  it's hard to have things like this break here and there.

**Comment by greglu** [comment]
Agreed that it's a mostly breaking change, which is why it's a major version upgrade. I couldn't come up with an easy way to migrate between the versions because of properties like `init_script_source`.

Sounds like locking to the old version would be better for you until you're able to update the resource usage.
